### ISSUE TITLE
20200815 -1 판다스 다루기 

### ISSUE THREAD

**Issue by dbgml4647** [dbgml4647/Meachine-Learning-Study#1]
`독립 변수` - 원인이 되는 변수 
`종속 변수` - 결과가 되는 변수 

프로그램을 이용 할 때는 독립 변수/종속 변수 분류 해야함 

표를 불러와서 해야함,,,, ! -> 판다스 다루기 

😀 `import pandas as pd`😀
tensorflow와는 무관한 도구임

우리가 사용하는 데이터는?

lemonade.csv
boston.csv
iris.csv

**Comment by dbgml4647** [comment]
> 데이터 준비하기 

1. 파일 읽어오기 : pd.read_csv
2. 모양 확인하기 :  print(데이터.shape)
3. 칼럼 선택하기 :  데이터[{'칼럼명1, '칼럼명2', '칼럼명3}]
4. 칼럼 이름 출력하기 : print(데이터.columns)
5. 맨 위 5개 관측치 출력하기 : 데이터.head()

**Comment by dbgml4647** [comment]
모델의 구조를 만드는것?

-> 근데 강의가 좀 어렵다?

ㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋ

1. 과거의 데이터를 준비한다
2. 모델의 구조를 만든다
3. 데이터로 모델을 학습합니다
4. 모델을 이용한다 ?


* 근데 진짜 피피티 잘한다,, 부럽다야


모델을 구성해야하는데?
- 뉴럴 네트워크로 만들어서.. -> 가장 간단하다고 한다 

![image](https://user-images.githubusercontent.com/34065333/90342952-4ac5d380-e047-11ea-956a-0a63245d9788.png)

model.fit -> 학습 시키는 간단한 코드 
`epochs = 몇 번 반복하는거 ? 의미`

**Comment by dbgml4647** [comment]
LOSS 

model.fit(독립, 종속, epochs=10)

10번 반복하는 하는거... 근데 loss 가 있네?

이게 뭐야?

= 학습이 얼마나 정답에 맞추고 있는가!

= (판매량 - 예측)의 제곱  의 평균
 
= loss가 적으면 좋은것이다!

= 여러 번 반복하면 반복할 수 록 적어진다 

= 😄
